dubbo-configcenter/dubbo-configcenter-nacos/pom.xml

```diff
@@ -41,6 +41,12 @@
         <dependency>
             <groupId>com.alibaba.nacos</groupId>
             <artifactId>nacos-client</artifactId>
+            <exclusions>
```

Comment:
It is necessary to exclude dependency in this?

Reply:
> It is necessary to exclude dependency in this?

It's not going to be used here, but it's a lightweight rule, and it's necessary to exclude